draft-ietf-ppm-dap.md

```diff
@@ -1764,9 +1773,28 @@ The Leader and Helper initialization behavior is detailed below.
 
 #### Leader Initialization {#leader-init}
 
+~~~ aasvg
+                     PrepareResp
+Report --> Continued -----+----> Continued
+                          |
+                          +----> FinishedWithOutbound
```

Comment:
```suggestion
                          +----> FinishedWithOutbound (*commit)
```


Isn't `FinishedWithOutbound` terminal? Sure, you send a message to the peer, but the aggregator that reaches this state doesn't expect to receive any further message.

Reply:
Not for the Leader! It's still possible for the Helper to reject at this point. The Helper always commits first.